### DISCUSSION TITLE
Implement XDG Base Directory Specification

### DISCUSSION THREAD

**Discussion by eszlari** [NVIDIA/open-gpu-kernel-modules#404] [Ideas]
At the moment these files/dirs are created in `$HOME`:
```
.nv/ComputeCache
.nvidia-settings-rc
```
It would be nice, if these could be moved to `.cache` and `.config`, following the XDG Base Directory Specification:

https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html

**Comment by aritger** [comment]
That's a fair suggestion, but open-gpu-kernel-modules isn't really the place for it--nothing in this repo interacts with those files.
